firecloud_project.py

```diff
@@ -39,10 +39,13 @@ def iprange(number):
     "dataflow.googleapis.com",
     "dataproc.googleapis.com",
     "genomics.googleapis.com",
-    "logging.googleapis.com",
     "storage-api.googleapis.com",
     "storage-component.googleapis.com",
-    "cloudkms.googleapis.com"
+    "cloudkms.googleapis.com",
```

Comment:
Nitpick / feel free to ignore: I think I'd originally tried to keep the list alphabetized so it's easier to quickly find a target API.